### ISSUE TITLE
[ENHANCEMENT] Improve JSON deserialize performance for large strings

### ISSUE THREAD

**Issue by lorenzobruni** [CSOIreland/Server-API-Library#7]
Improve performance by skipping deserialization for large strings.
This is possible by storing sub-MemCacheD records with a payload of type `String` rather than `MemCachedD_Value`

**Comment by lorenzobruni** [comment]
Add the following key to the Web.config and set the value of **API_MEMCACHED_MAX_SIZE** to the pre-set limit during the Server Side installation process (usually 128)

```xml
<!-- MemCacheD - Set the max size in MB before splitting a string record in sub-cache entries -->
<add key="API_MEMCACHED_MAX_SIZE" value="128" />
```

**Comment by lorenzobruni** [comment]
Implemented in Release 4.4.1 https://github.com/CSOIreland/Server-API-Library/releases/tag/4.4.1
